cmd/otelcol/config/collector/ecs_ec2_config.yaml

```diff
@@ -129,12 +129,20 @@ exporters:
     access_token: "${SPLUNK_ACCESS_TOKEN}"
     realm: "${SPLUNK_REALM}"
     correlation:
-  # Logs + Profiling
+  # Logs
   splunk_hec:
     token: "${SPLUNK_HEC_TOKEN}"
     endpoint: "${SPLUNK_HEC_URL}"
     source: "otel"
     sourcetype: "otel"
+    profiling_data_enabled: false
+  # Profiling
+  splunk_hec/profiling:
+    token: "${SPLUNK_ACCESS_TOKEN}"
+    endpoint: "${SPLUNK_INGEST_URL}/v1/log"
+    source: "otel"
+    sourcetype: "otel"
```

Comment:
are these used by the backend?

Reply:
Good question. I asked the profiling team

Reply:
While @vovencij is on vacation, I can confirm that we are not using these fields in any way on the server side.

Reply:
If these are optional fields and not used for profiling, I suggest we remove them from the default configuration of this dedicated profiling hec exporter.

Reply:
Removed these fields from the profiling exporters